packages/playground/src/app.tsx

```diff
@@ -102,7 +101,7 @@ const themes: PlaygroundProps['themes'] = {
     stylesheet: '',
     theme: FluentUIRCTheme,
   },
-  'material-ui-5': {
+  mui: {
     stylesheet: '',
     theme: MuiV5Theme,
```

Comment:
could be renamed to just MuiTheme now

Reply:
Good point, I will consider making a change to add `MuiTheme` and deprecating `MuiV5Theme`

Reply:
I just mean to change the import 

```
import { Theme as MuiV5Theme } from '@rjsf/mui';
```

to

```
import { Theme as MuiTheme } from '@rjsf/mui';
```

Reply:
Done